### DISCUSSION TITLE
Github Issues: Keyboard shortcut "C" not working to create new issues

### DISCUSSION THREAD

**Discussion by vhartikainen** [community/community#73964] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

To reproduce:
1. Head to : https://github.com/issues
2. Try to create new issue using keyboard shortcut: "C"

What happens:
Nothing.

Expected:
Open a form to create new issue.

Additional information:
As stated in the keyboard shortcuts (accessible via shortcut "?"), there should be a shortcut for creating issues with pressing of "C"

![image](https://github.com/community/community/assets/9887597/aa2b67e8-d5a5-4097-8c10-408c62976fe7)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by b8zeek** [comment, marked as answer]
Greetings Ville! 👋🏼 

That is not an issue buddy, I think you misunderstood the feature. `C` as a shortcut for creating a new issue actually works in the issues page of a repository, not on the list of all issues page. If you take a look at that page, there is no option to create a new issue because GitHub can't know which repository it should refer to.

Hope this clarifies it. Take care bro. 🍺 

🍀

**Reply by vhartikainen** [reply]
Sure does if it's designed that way :) But ... in projects, when you create a new issue, it's prompting for the repo where to create it :D On the other hand, for some reason, there's no kbd shortcut in projects to create new issues. I'd just rather be using keyboard, when creating a bunch of issues, so maybe there could be a shortcut added there?

**Reply by b8zeek** [reply]
From the projects page, `G` ➡️ `I` ➡️ `C` solves your issue. 😄 

🍀

**Reply by vhartikainen** [reply]
Well... not really, did you try it? 😄 As g -> i takes me to that generic issues list, at least from my project page. And that's the place where "C" doesn't work again 😉

**Reply by b8zeek** [reply]
I tested it from the repository Projects, works. From the profile's projects doesn't work because of the thing I mentioned above. And doesn't make much sense to enable creating an issue from the general issues as that page should have the scope to all projects and everything that comes with them. So creating a new issue from general issues page is not happening soon.

Take care.

🍀
